config/runtime.exs

```diff
@@ -0,0 +1,8 @@
+# General application configuration
+import Config
+
+config :lambda_ethereum_consensus, :mainnet_capella_presets,
+  YamlElixir.read_from_file!("config/presets/mainnet/capella.yaml") |> SpecTestUtils.parse_yaml()
```

Comment:
@MegaRedHand Here we do not need the parse yaml its only for the tests right ?

Reply:
Then i would let you or someone takeover considering I will go off for this week.